recipe_batches/recipe_batches.py

```diff
@@ -3,7 +3,17 @@
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+  recipeValue = recipe.values()
+  ingredientValue = ingredients.values()
+
+  if recipe.keys() == ingredients.keys():
+    for value1 in ingredientValue:
+      for value2 in recipeValue:
+
+        return math.floor((value1 / value2))
+    # return ingredients.values() / recipe.values() 
```

Comment:
1) I used the double For loop because I was trying to iterate over both sets of values and compare it but in hindsight it does not look like the right play & wasn't working as intended. I just thought it was because it passed the test. 

2) I thought it would return out of the nested loop & then start the initial loop over?

Reply:
1. Yeah a double for loop would be if for each value in the outer loop, you need to compare that value with each of the other values using the inner loop. But in this case, I think you just need to compare pairs of values (or in other words, you don't need to compare the qty of milk with the qty of butter, qty of flour, etc.).

2. `return` will actually end the function execution, and the value after the return is the value the function outputs. Again, `return` is like, "I'm done. This is what I got."